test.js

```diff
@@ -1,93 +1,301 @@
-var assert = require("assert");
-var generate = require("bit-docs-generate-html/generate");
-var path = require("path");
-
-var Browser = require("zombie"),
-	connect = require("connect");
-
-
-var find = function(browser, property, callback, done){
-	var start = new Date();
-	var check = function(){
-		if(browser.window && browser.window[property]) {
-			callback(browser.window[property]);
-		} else if(new Date() - start < 2000){
-			setTimeout(check, 20);
-		} else {
-			done("failed to find "+property);
+var path = require('path'),
+	rmrf = require('rimraf'),
+	assert = require('assert'),
+	Browser = require('zombie'),
+	express = require('express'),
+	generate = require('bit-docs-generate-html/generate');
```

Comment:
Well, the use of tabs is something I picked from other parts of the `bit-docs` codebase, and it is my understanding that it is @justinbmeyer's preference. I personally prefer spaces, but technically these do line up properly if you have your editor configured properly:

![image](https://cloud.githubusercontent.com/assets/990216/25560391/4da9ccb8-2d18-11e7-88ec-703ff4de08d2.png)

The spacing on GitHub's website could be resolved by adding an `.editorconfig`. 

I thought tabs were desired, and that's why I used them here. I'm fine with changing to using `var` on each declaration, or replacing tabs with spaces, or adding an `.editorconfig`.

At the end of the day, these tests do their job, and that's most important! I'll leave it how it is for now, as eventually I imagine we'll run everything through a code formatter anyways.